package/kedro_viz/models/experiment_tracking.py

```diff
@@ -2,16 +2,23 @@
 tracking datasets."""
 # pylint: disable=too-few-public-methods,protected-access,missing-class-docstring,missing-function-docstring
 import logging
+import base64
 from dataclasses import dataclass, field
 from enum import Enum
 from typing import Any, Dict
 
 from kedro.io import AbstractVersionedDataSet, Version
+from kedro.io.core import get_filepath_str
+
 from sqlalchemy import Column
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.sql.schema import ForeignKey
 from sqlalchemy.types import JSON, Boolean, Integer, String
 
+from kedro.extras.datasets.plotly import JSONDataSet as PlotlyJSONDataSet
+from kedro.extras.datasets.plotly import PlotlyDataSet
+from kedro.extras.datasets.matplotlib import MatplotlibWriter
```

Comment:
We can't have these as top-level imports because they won't work in a project that doesn't have the right optional dependencies (like matplotlib) installed.